src-docs/webpack.config.js

```diff
@@ -14,13 +14,13 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const CircularDependencyPlugin = require('circular-dependency-plugin');
 const babelConfig = require('./.babelrc.js');
 // const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
 
 const getPort = require('get-port');
-const deasync = require('deasync');
 
-const { NODE_ENV, CI, WEBPACK_DEV_SERVER } = process.env;
+const { NODE_ENV, CI, WEBPACK_SERVE } = process.env;
 
-const isDevelopment = WEBPACK_DEV_SERVER === 'true' && CI == null;
+const isDevelopment = WEBPACK_SERVE === 'true' && CI == null;
```

Comment:
`WEBPACK_DEV_SERVER` was changed to `WEBPACK_SERVE` (https://github.com/webpack/webpack-cli/pull/2027)